server/game/core/gameSteps/ActionWindow.js

```diff
@@ -156,7 +156,7 @@ class ActionWindow extends UiPrompt {
             // in the (unusual) case that both players pass without claiming initiative, phase ends and initiative stays where it is
             this.activePlayer.passedActionPhase = true;
             this.activePlayer.opponent.passedActionPhase = true;
-        } else if (this.activePlayer.opponent.passedActionPhase) {
+        } else if (this.activePlayer.opponent === this.game.initiativePlayer && this.game.isInitiativeClaimed) {
```

Comment:
I completely agree. After I pushed the PR, I was wondering how exactly we could make it so that the opponent consistently passes rather than checking if he has initiative.